### PR TITLE
Update deploy-gatsby.md 

### PR DESCRIPTION
**Pull request by lucastanz**
Added some s3 and cloudfront info

### CODE REVIEW

File: docs/docs/deploy-gatsby.md [hunk 1]
```diff
@@ -41,3 +41,8 @@ Uncaught Error: Minified React error #32; visit http://facebook.github.io/react/
 ```
 
 This is a new problem when dealing with static sites built with React.  React uses HTML comments to help identify locations of components that do not render anything.  If you are using a CDN that minifies your HTML, it will eliminate the HTML comments used by react to take control of the page on the client.
+
+## Hosting on amazon s3 and cloudfront
+If you decide to host your gatsbyjs site to s3 having cloudfront as CDN you should edit on the cloudfront panel the "Origin Domain Name" with the real URL of your s3 bucket: **examplewebsite.com.s3-website-eu-west-1.amazonaws.com** instead of the default one automatically suggested by amazon **examplewebsite.com.s3.amazonaws.com**.
```
Comment (KyleAMathews) [inline docs/docs/deploy-gatsby.md]: "gatsbyjs" should be "Gatsby". Capitalize other proper nouns e.g Amazon, Cloudfront, etc.

Comment (lucastanz) [inline docs/docs/deploy-gatsby.md]: ok Kyle, thanks :)

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 8152d7077f92bc9011308ed1030ff97960564fab

https://app.netlify.com/sites/using-glamor/deploys/598dbaee424ef229959cb04e

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 8a86525fc82d7208922239caabd7ff79b086db7c

https://app.netlify.com/sites/using-glamor/deploys/598dbca50752d016155039de

**Comment by sebastienfi** [comment]
:+1:
Thanks for helping others

**Comment by jlengstorf** [comment]
Hiya @lucastanz! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**